termsandconditions/views.py

```diff
@@ -44,6 +45,28 @@ def get_terms(self, kwargs):
             terms = TermsAndConditions.get_active_terms_not_agreed_to(self.request.user)
         return terms
 
+    def get_return_to(self, from_dict):
+        return_to = from_dict.get("returnTo", "/")
```

Comment:
Should we keep the `or "/"` part? It would overwrite an empty string passed like `{'returnTo': ''}`.